pom.xml

```diff
@@ -238,6 +216,7 @@
             <exclude>**/.idea/**</exclude>
             <exclude>LICENSE</exclude>
             <exclude>DISCLAIMER</exclude>
+            <exclude>DEPENDENCIES</exclude>
```

Comment:
nit 3: DEPENDENCIES is generated in the assembly, which will display a license warning when you build from the source dist otherwise